spec/40-other-protocols.md

```diff
@@ -1,13 +1,10 @@
 # Other protocols
 
-Trace Context propagation is crucial to enable distributed tracing scenarios
-for applications spans multiple components. Http is one of the communication
-protocols used for cross components correlation. Extensions of this
-specification as well as specifications defined by external organizations define
-the format of trace context serialization and
-deserialization for other protocols. Note, that those extensions may be at a
-different maturity level than this specification.
+While trace context is defined for HTTP the authors acknowledge it is also relevant
+for other protocols. Extensions of this specification define the format of trace
```

Comment:
why have you dropped "specifications defined by external organizations"? THis is the whole point of having a registry with external contributions